kiosk/settings/base.py

```diff
@@ -147,16 +149,18 @@
 # Static files (CSS, JavaScript, Images)
 # https://docs.djangoproject.com/en/2.1/howto/static-files/
 
+STATIC_ROOT = os.path.join(BASE_DIR, 'staticfiles') # The absolute path to the directory where collectstatic will collect static files for deployment.
 STATIC_URL = '/static/'
 
 STATICFILES_DIRS = [
     os.path.join(BASE_DIR, "static"),
 ]
 
+STATICFILES_STORAGE = 'whitenoise.storage.CompressedManifestStaticFilesStorage'
```

Comment:
This combines automatic compression with the caching behaviour provided by Django’s ManifestStaticFilesStorage backend.